extensions/smallrye-jwt/deployment/src/main/java/io/quarkus/smallrye/jwt/deployment/SmallryeJwtDevModeProcessor.java

```diff
@@ -55,7 +56,7 @@ public class SmallryeJwtDevModeProcessor {
      *
      * @throws NoSuchAlgorithmException if RSA-256 key generation fails.
      */
-    @BuildStep(onlyIfNot = { IsNormal.class })
+    @BuildStep(onlyIfNot = { IsNormal.class, NativeOrNativeSourcesBuild.class })
```

Comment:
I would expect the native build to run in normal mode? So I'm a bit surprised it makes a difference?

Reply:
my bet is NativeSourcesBuild part, but I can look more into it

Reply:
Having just `@BuildStep(onlyIfNot = { IsNormal.class, NativeSourcesBuild.class })` doesn't fix the issue

So my change is the minimal one.